app/src/main/java/ch/epfl/swissteam/services/ChatRelation.java

```diff
@@ -18,7 +18,7 @@
     private String secondUserId_;
     private String id_;
 
-    public ChatRelation(User firstUser, User secondUser ){
+    public ChatRelation(User firstUser, User secondUser ) {
         this();
```

Comment:
Sauf erreur, t'as pas besoin de mettre id_ = null dans le constructeur vide, vu que c'est une valeur par défaut, il le fait tout seul. Du coup le this() de cette méthode est pas nécessaire non plus
Je suis pas un grand fan d'utiliser les setters à la construction, mais au moins les checks sont faits à un seul endroit.
Pourrais-tu quand meme envisager de faire une assignation "standard", ca pourrait faire assez propre, du point de vue visibilité. T'en penses quoi?